lib/afc/streams.js

```diff
@@ -52,13 +54,15 @@ class AfcWritableFileStream extends stream.Writable {
     callback();
   }
 
-  async _destroy () {
+  async _destroy (err, cb) {
     if (this._destroyed) {
       return;
     }
     this._destroyed = true;
-    await this._afcService.closeFileHandle(this._fileHandle);
-    this.emit('close');
+    try {
+      await this._afcService.closeFileHandle(this._fileHandle);
+    } catch (ign) {}
+    cb(err);
```

Comment:
should we check if cb is a function here?

Reply:
The documentation says it will be always a function